setup.py

```diff
@@ -9,21 +9,84 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+
+import atexit
 import site
 import sys
 
 from setuptools import find_namespace_packages, setup
+from setuptools.command.install import install
 
 import versioneer
 
 # Workaround for editable installs with system's Python venv.
 #   error: can't create or remove files in install directory
 # (https://github.com/pypa/pip/issues/7953#issuecomment-645133255)
 site.ENABLE_USER_SITE = "--user" in sys.argv[1:]
+class PostInstallCommand(install):
+    """Contains post install actions."""
+
+    def __init__(self, *args, **kwargs):
+        super(PostInstallCommand, self).__init__(*args, **kwargs)
+        atexit.register(PostInstallCommand.patch_holoscan)
+
+    @staticmethod
+    def patch_holoscan():
+        """ Patch Holoscan for its known issue of missing one import."""
+
+        import importlib.util
+        from pathlib import Path
+
+        def needed_to_patch():
+            from importlib.metadata import version
+            try:
+                version = version("holoscan")
+                # This issue exists in the following versions
+                if "2.7" in version or "2.8" in version:
+                    print("Need to patch holoscan v2.7 and 2.8.")
+                    return True
+            except Exception:
+                pass
+
+            return False
+
+        if not needed_to_patch():
+            return
+
+        print("Patching holoscan as needed...")
+        spec = importlib.util.find_spec("holoscan")
+        if spec:
```

Comment:
Do we need to warn if the module cannot be found somehow? Highly unlikely I guess?

Reply:
Since holoscan is a required dependency of monai-deploy-app-sdk, I'd think pip would have failed the install if holoscan is not present (well, the holoscan package could be uninstalled afterwards, but then it is post post-install actions)